javascript/README.md

```diff
@@ -402,3 +402,48 @@ Prefer the short, more succinct version.
 Array.prototype.forEach.call(arguments, function(arg) { console.log(arg); });
 ```
 
+##Prefer `self` to `bind(this)`
```

Comment:
Should we be more strong with the wording here?  "Always use" vs "Prefer".  Can anyone think of a reason to use `.bind()` where `self` wouldn't be just as easy/good?